From ec240a68c413c4ac7c1929436f6443e60866548d Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Wed, 26 Apr 2017 14:49:24 -0400
Subject: [PATCH] dconf defaults: Restore session-name override

This was dropped in f66cdfcb but GDM won't start when the
default session-name is set to something else

https://bugzilla.gnome.org/show_bug.cgi?id=781793
---
 data/dconf/defaults/00-upstream-settings | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/data/dconf/defaults/00-upstream-settings b/data/dconf/defaults/00-upstream-settings
index 0f014cef..c0b3cc66 100644
--- a/data/dconf/defaults/00-upstream-settings
+++ b/data/dconf/defaults/00-upstream-settings
@@ -6,6 +6,9 @@
 #       dconf update
 #
 
+[org/gnome/desktop/session]
+session-name='gnome'
+
 [org/gnome/desktop/background]
 show-desktop-icons=false
 
-- 
2.11.0

